src/open_producten/producttypen/serializers/link.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+
+from open_producten.producttypen.models import Link, ProductType
+
+
+class LinkSerializer(serializers.ModelSerializer):
+    product_type_id = serializers.PrimaryKeyRelatedField(
+        source="product_type", queryset=ProductType.objects.all()
+    )
+
+    class Meta:
+        model = Link
+        exclude = ("product_type",)
```

Comment:
```suggestion
        fields = ("id", "naam", "url", "product_type_id")
```

Please use explicit field declarations instead of include.